apps/reflect/src/components/app/AppBar/index.tsx

```diff
@@ -62,26 +54,19 @@ const ResponsiveAppBar = ({
           backgroundColor: alpha(theme.palette.background.paper, 0.85),
         }}
         elevation={2}>
-        {/*open={isDrawerOpen} */}
         <Container>
           {/* <Container maxWidth="xl">*/}
           <Toolbar disableGutters>
             <AppBarTitle />
-
             <AppBarButton
               title="Go to Interface"
               pathHref={interfaceURL}
               ariaLabel="go to interface"
               drawerHandler={drawerHandler}>
               <MinimizeIcon />
             </AppBarButton>
-
-            <GetTwitchBotButton />
-            <GetDiscordBotButton />
-            <CommandListButton />
-
+            <CommandListButton onClick={() => setDrawerIsOpen(!drawerIsOpen)} />

```

Comment:
I think we should change the name of this component to `DrawerButton` or something.